### PR TITLE
[All Bundles] Added PostgreSQL support

### PR DESCRIPTION
**Pull request by igst**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -

Based on https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/1778 and https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/1810

This PR makes it possible to use Kunstmaan Bundles CMS with a PostgreSQL database.

### CODE REVIEW

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* It seems that you should have submitted to the latest minor branch.
* This PR seems to need a milestone of a patch release.

File: src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/article/Repository/PageRepositoryPartial.php.twig [hunk 1]
```diff
@@ -41,8 +41,10 @@ class {{ entity_class }}PageRepository extends AbstractArticlePageRepository
...
             ->innerJoin('KunstmaanNodeBundle:Node', 'n', 'WITH', 't.node = n.id')
-            ->where('t.online = 1')
-            ->andWhere('n.deleted = 0')
+            ->where('t.online = :online')
+            ->setParameter('online', true)
+            ->andWhere('n.deleted = :deleted')
+            ->setParameter('deleted', false)
             ->andWhere('v.refEntityName = :refname')
             ->orderBy('a.date', 'DESC')
             ->setParameter('refname', "{{ namespace | replace({'\\': '\\\\'}) }}\\Entity\\Pages\\{{ entity_class }}Page");
```
Comment (igst) [inline src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/article/Repository/PageRepositoryPartial.php.twig:50]: //// is only required in LIKE clause and should not work with "="

File: src/Kunstmaan/AdminBundle/Helper/Security/Acl/AclHelper.php [hunk 2]
```diff
@@ -166,40 +166,41 @@ private function getPermittedAclIdsSQLForUser(Query $query)
...
+INNER JOIN acl_classes c ON c.id = o.class_id
+LEFT JOIN acl_entries e ON (
     e.class_id = o.class_id AND (e.object_identity_id = o.id
     OR {$aclConnection->getDatabasePlatform()->getIsNullExpression('e.object_identity_id')})
 )
-LEFT JOIN {$databasePrefix}acl_security_identities s ON (
+LEFT JOIN acl_security_identities s ON (
 s.id = e.security_identity_id
 )
 WHERE c.class_type = {$rootEntity}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Helper/Security/Acl/AclHelper.php:206]: Do we need quotes here? https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/1778/files#diff-2ae42119f573a6a61e26009641aa186d97e0f77b951dc02385d67d9daa075854R193

File: src/Kunstmaan/AdminBundle/Helper/Security/Acl/AclNativeHelper.php [hunk 3]
```diff
@@ -99,40 +99,41 @@ public function apply(QueryBuilder $queryBuilder, PermissionDefinition $permissi
...
+INNER JOIN acl_classes c ON c.id = o.class_id
+LEFT JOIN acl_entries e ON (
     e.class_id = o.class_id AND (e.object_identity_id = o.id
     OR {$aclConnection->getDatabasePlatform()->getIsNullExpression('e.object_identity_id')})
 )
-LEFT JOIN {$databasePrefix}acl_security_identities s ON (
+LEFT JOIN acl_security_identities s ON (
 s.id = e.security_identity_id
 )
 WHERE c.class_type = {$rootEntity}
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/Helper/Security/Acl/AclNativeHelper.php:139]: Same question as comment above

**Review by acrobat — COMMENTED** [review]
Few questions. I will also test this PR on some actually projects to avoid any breaks there

**Comment by acrobat** [comment]
I'm closing the PR in favor of #2849, the implentation will be finished there. Thanks for your work on this @igst!

**Pull request closed without merging**